.gitignore

```diff
@@ -27,3 +27,4 @@ paper
 # Ignore IDE project files
 .idea/
 .vscode
+.vs
```

Comment:
It's generally best to add these pointers to your own ["global" gitignore](https://sebastiandedeyne.com/setting-up-a-global-gitignore-file/), rather than to every project you work on. That way, it never accidentally slips in without you realising it!